aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp

```diff
@@ -174,8 +175,9 @@ at::Tensor& embedding_bag_4bit_impl(
 
   const auto weight_sizes = weight.sizes();
   const int64_t weight_size = weight_sizes[1];
+  int NUM_ELEM_PER_BYTE = 8 / bit_width;
```

Comment:
~~should we add assert that `bit_width` must be > 0 and <= 8, and can only be multiples of 2? So the `bit_width` can only be 2, 4, or 8~~

nvm, noticed the assert is done below